docs/pages/eas/webhooks.md

```diff
@@ -18,16 +18,18 @@ Webhooks function almost exactly the same for both EAS Build and the classic `ex
 </p>
 </details>
 
-After running the command, you'll have to provide the webhook URL (or specify it with the `--url` flag) that handles HTTP POST requests. Additionally, you'll have to input a webhook signing secret, if you have not already provided it with the `--secret` flag. It must be at least 16 characters long, and it will be used to calculate the signature of the request body which we send as the value of the `expo-signature` HTTP header. You can use the signature to verify a webhook request is genuine (example code below).
+After running `eas webhook:create`, you'll be prompted to choose the webhook event type (unless you provide the `--event BUILD|SUBMIT` parameter). Next, provide the webhook URL (or specify it with the `--url` flag) that handles HTTP POST requests. Additionally, you'll have to input a webhook signing secret, if you have not already provided it with the `--secret` flag. It must be at least 16 characters long, and it will be used to calculate the signature of the request body which we send as the value of the `expo-signature` HTTP header. You can use the signature to verify a webhook request is genuine (example code below).
 
-We call your webhook using an HTTP POST request and we pass data in the request body. EAS sends the build data as a JSON object. The most notable fields are:
-
-- `id` - the unique ID of your build
-- `status` - a string specifying whether your build has finished successfully (can be either `finished` or `errored`)
-- `artifacts.buildUrl` - the URL to the build artifact (only included if `status === 'finished'`)
+EAS calls your webhook using an HTTP POST request. All the data is passed in the request body. EAS sends the data as a JSON object. The most notable fields are:
```

Comment:
"The most notable fields are:"

it looks like we aren't listing any fields here anymore. we should probably add some documentation for the fields for both build and submit webhooks